packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -23,6 +28,8 @@ const propTypes = {
   parentId: PropTypes.string.isRequired,
   /** Progress title. */
   title: PropTypes.string,
+  /** Label to indicate what progress is shwoing. */
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.number]),
```

Comment:
same comments as above
```suggestion
  label: PropTypes.node,
```